src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -294,6 +294,71 @@ public Mnemonic CreateWallet(string password, string name, string passphrase, Mn
             return mnemonic;
         }
 
+        /// <inheritdoc />
+        public string SignMessage(string password, string walletName, string externalAddress, string message)
+        {
+            Guard.NotEmpty(password, nameof(password));
+            Guard.NotEmpty(walletName, nameof(walletName));
+            Guard.NotEmpty(message, nameof(message));
+            Guard.NotEmpty(externalAddress, nameof(externalAddress));
+
+            string signature = string.Empty;
+
+            // Load the file from the local system.
+            Wallet wallet = this.fileStorage.LoadByFileName($"{walletName}.{WalletFileExtension}");
```

Comment:
This is inside the wallet manager so wallets are already loaded, there is likely no need load a wallet from file just look it up in one of the built in method.